src/main/java/de/uhd/ifi/se/decision/management/jira/view/decisiontable/DecisionTable.java

```diff
@@ -132,12 +135,14 @@ public void getArgumentCriteria(Argument argument, List<DecisionTableElement> cr
 		// persistenceManager
 		KnowledgeElement rootElement = persistenceManager.getKnowledgeElement(argument.getId(),
 				argument.getDocumentationLocation());
-		Set<Link> outgoingLinks = this.graph.outgoingEdgesOf(rootElement);
+		Set<Link> linksOfArgument = this.graph.edgesOf(rootElement);
 
-		for (Link currentLink : outgoingLinks) {
-			KnowledgeElement elem = currentLink.getTarget();
+		for (Link currentLink : linksOfArgument) {
+			KnowledgeElement elem = currentLink.getOppositeElement(rootElement);

```

Comment:
getOpposite() could be null 
![image](https://user-images.githubusercontent.com/33954695/93567674-4c105600-f98f-11ea-822f-d4c62b0e1034.png)